### PR TITLE
get comment with review ID

### PR DESCRIPTION
**Pull request by zaghloulegy**
i create model  and controller to be able to get a comment with review Id

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -193,4 +193,35 @@ describe("/api/reviews", () => {
...
+        body: { comments },
+      } = await request(app).get("/api/reviews/2/comments").expect(200);
+      expect(Array.isArray(comments)).toBe(true);
+    });
+    test("200: returns an array of comments in correct format", async () => {
+      const {
+        body: { comments },
+      } = await request(app).get("/api/reviews/2/comments").expect(200);
+
+      comments.forEach((comment) => {
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:211]: What happens with an empty array? This test would show a false positive.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -193,4 +193,35 @@ describe("/api/reviews", () => {
...
+          body: expect.any(String),
+        });
+      });
+    });
+    test("200: returns an empty array if review has no comments", async () => {
+      const {
+        body: { comments },
+      } = await request(app).get("/api/reviews/1/comments").expect(200);
+      expect(Array.isArray(comments)).toBe(true);
+    });
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:225]: You are missing tests for error handling on this endpoint.

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
Please complete error handling for this ticket then resubmit your PR to the helpdesk

File: __tests__/app.test.js (hunk 2 above)
Comment (zaghloulegy) [reply __tests__/app.test.js:225]: i just noted now 
thanks verity

**Review by elijaharley — APPROVED** [review]
Great stuff Mohamed - keep up the good work

**Pull request merged**